config/jobs/kubernetes/sig-release/release-branch-jobs/1.15.yaml

```diff
@@ -204,6 +204,60 @@ periodics:
   - 'perfDashPrefix: gce-100Nodes-1.15'
   - 'perfDashJobType: performance'
   - 'perfDashBuildsCount: 500'
+- annotations:
+    fork-per-release-cron: 0 21 * * *
```

Comment:
Should be removed?

Reply:
after each release, the oldest one is removed, the other are rotated.
Rotation pull first item from fork-per-release-cron and replaces cron variable with it.